src/main/java/com/pagsestagio/movieapi/controller/FilmeController.java

```diff
@@ -1,85 +1,82 @@
 package com.pagsestagio.movieapi.controller;
 
+import com.pagsestagio.movieapi.controller.resposta.FilmeResposta;
+import com.pagsestagio.movieapi.controller.resposta.FilmeRespostaErroRetornaMensagem;
+import com.pagsestagio.movieapi.controller.resposta.FilmeRespostaSucessoRetornaFilme;
+import com.pagsestagio.movieapi.controller.resposta.FilmeRespostaSucessoRetornaLista;
 import com.pagsestagio.movieapi.model.*;
```

Comment:
Acredito que seja interessante não importar tudo com *, nem digo por performance ou algo do tipo, até pq aparentemente o bytecode gerado posteriormente parece o mesmo, mas sobre a questão de importar apenas as classes que você realmente irá usar ajuda e facilita de alguma forma outra pessoa que for mexer no seu código. Além disso o import .* pode trazer complicações entre nome de classes, por exemplo, java.util.* \ java.sql.*, esses dois possuim dentro do seu pacote a classe Date, o que deixa ambíguo para o compilador fazendo-o não conseguir resolver sozinho.

Reply:
Fiquei um pouco confuso, poderia dar um exemplo de como seria essa ideia?

Reply:
Claro, posso sim!
Repara que o seu import do FilmeResposta é _import com.pagsestagio.movieapi.controller.resposta.FilmeResposta;_ isso significa que dentro do pacote import com.pagsestagio.movieapi.controller.resposta existe uma classe chamada FilmeResposta. Além do FilmeResposta, existe outras classes desse pacote que você está usando, como por exemplo o FilmeRespostaErroRetornaMensagem entre outros.

Caso você colocasse o import com.pagsestagio.movieapi.controller.resposta.*; você poderia utilizar todos esses imports também, mesmo sem declarar explícito o uso deles. Porém como eu disse anteriormente, isso pode causar alguns problemas. Então o interessante utilizar seria apenas o _import_ das classes que você realmente está utilizando e não necessáriamente todas classes do pacote.

Nesse caso específico, o _pacote com.pagsestagio.movieapi.model_ está sendo importado inteiro, com todas suas classes do pacote, para a classe FilmeController.
Para resolver isso é simples hehe, basta você importar somente o que você realmente está usando.
Algo como _com.pagsestagio.movieapi.model.ModeloExemplo_